packages/riipen-ui/src/components/Typography.jsx

```diff
@@ -324,6 +331,13 @@ class Typography extends React.Component {
             font-weight: ${theme.typography.fontWeight.regular};
           }
 
+          .letter-spacing-1 {
+            letter-spacing: 1px;
```

Comment:
You could maybe take it one step further and do 
.letter-spacing {
 letter-spacing: ${letterSpacing}px;
}

And keep the boolean at line 170 but do
letterSpacing ? 'letter-spacing' : null

Reply:
That sounds good thanks for the suggestion! 👏🏻